subsys/mpsl/cx/bluetooth/mpsl_cx_1w_bluetooth.c

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
This does not belong in the DTS commit, since it's really initializing the whole module. The PR's commit history is a bit hard to understand/review in general. 

I'd prefer if you put all the changes for enabling 1-wire coex into 1 commit (you could even put the sample in), and renaming of the kconfig/sources into separate commits.
DTS could be put in its own commit but then please don't include initialization code with it (ie. only have the yaml files).

Reply:
updated